### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by buzzlightyear1309** [Wassimulator/CactusViewer#39] [Q&A] [answered]
I really love CactusViewer so far, thanks for the hard work!
I have some feature request if possible, that I think can add some good value.
Could there be a toggle in the config to always show the navigation controls, not just on hover?
Also a toggle to start in full screen mode?
And im not sure if the following will be too complicated. But would it be possible to have the background transparent (darkened), so that we can still see the desktop/whatever behind it. Like Picasa Photo Viewer

**Comment by Wassimulator** [comment, marked as answer]
added the requested toggles for fullscreen and always show GUI in the latest release in v2.1.6

as for the transparent background, that needs more work, please open a separate ticket with that feature request.

**Reply by buzzlightyear1309** [reply]
Thanks so so much! Will open another ticket for that
